daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/schematron/TestEmbedded.scala

```diff
@@ -40,99 +41,111 @@ class TestEmbedded {
+    }(ExitCode.Success)
   }
 
   @Test def extends3(): Unit = {
     val schema = path("daffodil-schematron/src/test/resources/xsd/extends-1.dfdl.xsd")
 
-    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") { cli =>
-      cli.send(";;smith", inputDone = true)
-      cli.expectErr("Validation Error: first is blank")
-    } (ExitCode.ParseError)
+    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") {
+      cli =>
+        cli.send(";;smith", inputDone = true)
+        cli.expectErr("Validation Error: first is blank")
+    }(ExitCode.ParseError)
   }
 
   @Test def extends4(): Unit = {
     val schema = path("daffodil-schematron/src/test/resources/xsd/extends-1.dfdl.xsd")
 
-    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") { cli =>
-      cli.send("bob;l;", inputDone = true)
-      cli.expectErr("Validation Error: last is blank")
-    } (ExitCode.ParseError)
+    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") {
+      cli =>
+        cli.send("bob;l;", inputDone = true)
+        cli.expectErr("Validation Error: last is blank")
+    }(ExitCode.ParseError)
   }
 
   @Test def extends5(): Unit = {
     val schema = path("daffodil-schematron/src/test/resources/xsd/extends-1.dfdl.xsd")
 
-    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") { cli =>
-      cli.send(";l;", inputDone = true)
-      cli.expectErr("Validation Error: last is blank")
-      cli.expectErr("Validation Error: first is blank")
-    } (ExitCode.ParseError)
+    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") {
+      cli =>
+        cli.send(";l;", inputDone = true)
+        cli.expectErr("Validation Error: last is blank")
+        cli.expectErr("Validation Error: first is blank")
+    }(ExitCode.ParseError)
   }
 
   @Test def testWithNs1(): Unit = {
     val schema = path("daffodil-schematron/src/test/resources/xsd/with-ns-1.dfdl.xsd")
 
-    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") { cli =>
-      cli.send("0;1", inputDone = true)
-      cli.expect("</myns:interval>")
-    } (ExitCode.Success)
+    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") {
+      cli =>
+        cli.send("0;1", inputDone = true)
+        cli.expect("</myns:interval>")
+    }(ExitCode.Success)
   }
 
   @Test def testWithNs2(): Unit = {
     val schema = path("daffodil-schematron/src/test/resources/xsd/with-ns-1.dfdl.xsd")
 
-    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") { cli =>
-      cli.send("2;1", inputDone = true)
-      cli.expectErr("Validation Error")
-    } (ExitCode.ParseError)
+    runCLI(args"""parse --validate schematron="${jsonEscape(schema.toString)}" -s $schema""") {
+      cli =>
```

Comment:
I think this is another example about this being very aggressive about line lengths which ultimately hurts readability.